news/create-new-attrs-directly.rst

```diff
@@ -0,0 +1,31 @@
+New features
+------------
+
+* <news item>
+
+Deprecations
+------------
+
+* <news item>
+
+Exposing HDF5 functions
+-----------------------
+
+* <news item>
+
+Bug fixes
+---------
+
+* New attributes are created directly instead via a temporary
+attribute with subsequent renaming. This fixes overwriting
+attributes with ``track_order=True``.
```

Comment:
```suggestion
* New attributes are created directly, instead of via a temporary
  attribute with subsequent renaming. This fixes overwriting
  attributes with ``track_order=True``.
```